content/_data/home.yml

```diff
@@ -0,0 +1,33 @@
+features:
+  - title: OPEN AND FREE
+    text:  "[Free and open standard](/specification/)"
+    icon: freeandopen.png
+    icon-alt: Free and Open Icon - an open circular lock composed of a circle, keyhole and a open latch 
+  - title: COMMUNITY FIRST
+    text: "[Community](/community/) is a core principle of the CWL Project"  
+    icon: noun_Community_943325.svg
+    icon-alt: Community icon four people connected via a circle 
+  - title: INTEROPERABILITY AND PORTABILITY
+    icon: noun_Interoperability_181229.svg
+    icon-alt: Interoperability icon - a triangle composed of arrows, with a triangle, square, and circle as the 3 points
+    text: Portable and interoperable across a [variety of software and deployment environments](/implementations)
+  - title: VENDOR NEUTRALITY
+    icon: noun_Scale_380185.svg
+    icon-alt: Vendor Neutrality icon - an evenly weighted mechanical scale
+    text: Developed by a multi-vendor working group of [organizations](/governance/#participating-organizations/) and [individuals](/governance/#individual-contributors/)
```

Comment:
These links need fixing

Reply:
I don't know which links you mean.

Reply:
Fixed in commit: [56b8f1e123cf29db5c533ba284713b3e9ac6a0fd ](https://github.com/common-workflow-language/cwl-website/pull/75/commits/56b8f1e123cf29db5c533ba284713b3e9ac6a0fd)